paddle/fluid/framework/ir/graph_helper.cc

```diff
@@ -163,12 +166,13 @@ size_t GraphNum(const Graph &graph) {
     graph_nodes.emplace_back(g_nodes);
   }
 
-  if (VLOG_IS_ON(10)) {
-    VLOG(10) << "graph_num: " << graph_nodes.size();
+  if (FLAGS_display_n_nodes_of_sub_graph) {
```

Comment:
Can put this in graphviz pass to output more than one file if `graph_num > 1`?

Reply:
Thanks, I will fix it in next commit.